src/components/protection/AddProtectionSingle.vue

```diff
@@ -192,8 +192,7 @@ export default class AddProtectionSingle extends BaseComponent {
   tknSpaceAvailableLTOne: boolean = false;
```

Comment:
Larger than one is irrelevant to the UI layer, we just want to identify if a feature or piece of information is available or not. 
Would ```tknSpaceIsAvailable``` make sense here?